.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
   - pip install .
   - pip install -r test-requirements.txt
   - pip install -r requirements.txt
+  - pip install -r examples/django_app/requirements.txt
```

Comment:
This actually shouldn't be installed during testing. The first line (`pip install .`) installs the ChatterBot codebase for the current commit being tested. `pip install -r examples/django_app/requirements.txt` will install the current version of ChatterBot from PyPi.

Additionally, the tests also test on different versions of Django to ensure compatibility for each supported version. Installing from the example app requirements would alter the version of Django that gets installed to the test environment on Travis CI.